LICENSE.mustache

```diff
@@ -29,8 +29,9 @@ latest contribution.
 
 To give me credit, give users of your software notice that I
 contributed to this software, and indirectly to your software,
-and make sure that it is easy for users to find when using your
-software. For example, if users access your software via the
+including the texts of the copyright and source notices above.
```

Comment:
Should this be “**by** including”?

Reply:
Might word it better, but my initial answer is _no_. Users need to do all of these:

1. Tell users I contributed to this software.
2. Tell users I contributed indirectly to their software.
3. Include my copyright and source notices.

I don't think just copies of the copyright and source notices effectively achieves credit. Credit for the open software needs to be linked to credit for the software users are actually using.

Reply:
OK, then I definitely found the late-floating ing clause confusing. I didn’t parse this as “the three things you need to do”. I think partly it’s the “and” earlier confusing things.

I think it may be clearer to reorder as: “to give credit, include these things and give users of your software notice that I both contributed to this software and also indirectly to your software.”

Even still, I’m not terribly clear on what a Notices page that credits, say, 5 different projects using this license would look like in terms of making it a readable structure. Probably something like, “The following persons contributed indirectly to this software via their work on the listed software:

* Jane Doe, SomeLibrary, link to license text anchor further down the page goes here”

That is, frontload the in/direct credit bits and stuff the license details below the fold. So I guess I can figure it out once I know those three separate things are needed.

Reply:
I am about to push a couple more commits that I think help clarify the requirements.

As for your notes on what the actual credits page would look like, what you described matches what I had in mind.